gtfs/spec/en/reference.md

```diff
@@ -176,7 +176,7 @@ File: **Required**
 |  Field Name | Required | Details |  |
 |  ------ | ------ | ------ | ------ |
 |  trip_id | **Required** | The **trip_id** field contains an ID that identifies a trip. This value is referenced from the [trips.txt](#tripstxt) file. |  |
-|  arrival_time | **Required** | The **arrival_time** specifies the arrival time at a specific stop for a specific trip on a route. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service date. For times occurring after midnight on the service date, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.  If this stop isn't a time point, use an empty string value for the **arrival_time** and **departure_time** fields. Stops without arrival times will be scheduled based on the nearest preceding timed stop. To ensure accurate routing, please provide arrival and departure times for all stops that are time points. Do not interpolate stops.  You must specify arrival and departure times for the first and last stops in a trip.  Times must be eight digits in HH:MM:SS format (H:MM:SS is also accepted, if the hour begins with 0). Do not pad times with spaces. The following columns list stop times for a trip and the proper way to express those times in the **arrival_time** field: |  |
+|  arrival_time | **Required** | The **arrival_time** specifies the arrival time at a specific stop for a specific trip on a route. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service date. For times occurring after midnight on the service date, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.  If this stop isn't a time point, use an empty string value for the **arrival_time** and **departure_time** fields or provide an interpolated time, indicated by providing a value of 0 for the **timepoint** field. Stops without arrival times will be scheduled based on the nearest preceding timed stop. To ensure accurate routing, please provide arrival and departure times for all stops that are time points. You must specify arrival and departure times for the first and last stops in a trip.  Times must be eight digits in HH:MM:SS format (H:MM:SS is also accepted, if the hour begins with 0). Do not pad times with spaces. The following columns list stop times for a trip and the proper way to express those times in the **arrival_time** field: |  |
```

Comment:
>Stops without arrival times will be scheduled based on the nearest preceding timed stop

I vote for removing this.  This is the behavior of one specific GTFS-consuming application (Google Maps), and other applications may have different (and potentially improved) ways to interpolate/extrapolate times.  I believe it's also possible for the first stop in the trip to have `timepoint=0` (I recall someone having this case), in which case the above method isn't possible.